cypress/support/helper.js

```diff
@@ -111,8 +110,23 @@ export function pressTABKey(count) {
 }
 
 export function pressTABKeyInNoIframe(count) {
+  let body = cy.get("body");
```

Comment:
1) I can't seem to move out the body variable. I get an error saying that ` Cannot call cy.get() outside a running test`. I think I will need some help to achieve it, but agree it would be a good idea